### ISSUE TITLE
not an issue - notion

### ISSUE THREAD

**Issue by hamlct** [tthn0/Spotify-Readme#33]
Just wondering if anyone has added this to a notion page as a widget? Would like some help if its possible

**Comment by tthn0** [comment]
Yes, you can add it to a notion page. Just create a new block and select "Image" . You can then paste the link to your deployed instance. You'll also need to resize the image block in notion to get it to fit.
